services/dy-mapcore-widget/README.md

```diff
@@ -0,0 +1,15 @@
+# Simcore - Mapcore's flaftmap widget service
+
+TODO
+
+## local development
+
+1. TODO
+
+### dependencies
+
+TODO
+
+## production
+
+1. TODO

```

Comment:
extra instructions would be helpful here to allow others to make changes

Reply:
Done